<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainwidget">
   <widget class="QPushButton" name="generate_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>85</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>339</width>
      <height>192</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="audio_label">
       <property name="text">
        <string>Audio file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="timing_label">
       <property name="text">
        <string>Timing file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="epub_label">
       <property name="text">
        <string>EPUB file</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="css_label">
       <property name="text">
        <string>CSS file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="timing_filedialog">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="range_label">
       <property name="text">
        <string>Page range</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="4">
      <widget class="QLineEdit" name="range_text">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="audio_filedialog">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="css_filedialog">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="epub_filedialog">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="4">
      <widget class="QLineEdit" name="epub_text"/>
     </item>
     <item row="1" column="2" colspan="4">
      <widget class="QLineEdit" name="audio_text"/>
     </item>
     <item row="2" column="2" colspan="4">
      <widget class="QLineEdit" name="timing_text"/>
     </item>
     <item row="3" column="2" colspan="4">
      <widget class="QLineEdit" name="css_text">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEPUB_Readalong_Generator">
    <property name="title">
     <string>EPUB Readalong Generator</string>
    </property>
   </widget>
   <addaction name="menuEPUB_Readalong_Generator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
